### PR TITLE
I14467 update canton overview docs

### PR DESCRIPTION
**Pull request by matthiasS-da**

### CODE REVIEW

File: docs/2.8.0/docs/canton/architecture/overview.rst [hunk 1]
```diff
@@ -377,22 +334,23 @@ is assigned by the sequencer when registering the confirmation request (initial
 of the transaction).
 
 There is only a bounded relationship between these times, ensuring that the `ledger time` must be
-in a pre-defined bound around the `record time`. The tolerance (``max_skew``) is defined on the domain
+in a pre-defined bound around the `record time`. The tolerance is defined on the domain
 as a domain parameter, known to all participants
```
Comment (marciw-da) [inline docs/2.8.0/docs/canton/architecture/overview.rst:338]:
```suggestion
as a domain parameter, known to all participants:
```

File: docs/2.8.0/docs/canton/architecture/overview.rst [hunk 2]
```diff
@@ -509,23 +467,30 @@ We now list the high-level requirements on the sequencer.
 
 **Ordering:** The sequencer provides a `global total-order
 multicast <http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.85.3282&rep=rep1&type=pdf>`__
-where messages are uniquely time-stamped and the global ordering is
-derived from the timestamps. Instead of delivering a single message, the
-sequencer provides message batching, that is, a
-list of individual messages are submitted. All these messages get the
-timestamp of the batch they are contained in. Each message may
-have a different set of recipients; the messages in each recipient's batch
+where envelopes are uniquely time-stamped and the global ordering is
```
Comment (marciw-da) [inline docs/2.8.0/docs/canton/architecture/overview.rst:470]:
```suggestion
where envelopes are uniquely timestamped and the global ordering is
```

File: docs/2.8.0/docs/canton/architecture/overview.rst [hunk 3]
```diff
@@ -509,23 +467,30 @@ We now list the high-level requirements on the sequencer.
...
+derived from the timestamps. Instead of delivering a single envelope, the
+sequencer provides batching, that is, a
+list of individual envelopes are submitted. All these envelopes get the
```
Comment (marciw-da) [inline docs/2.8.0/docs/canton/architecture/overview.rst:473]:
```suggestion
list of individual envelopes are submitted. All of these envelopes get the
```

File: docs/2.8.0/docs/canton/architecture/overview.rst [hunk 4]
```diff
@@ -509,23 +467,30 @@ We now list the high-level requirements on the sequencer.
...
 A recipient only learns the identities of recipients
-on a particular message from a batch if it is itself a recipient of that
-message.
+on a particular envelope from a batch if it is itself a recipient of that
+envelope.
+
+.. note:: In the implementation, the recipients of an envelope are not a set of members (as indicated above),
+   but a forest of sets of members.
+   A member receives an envelope if it appears somewhere in the recipient forest.
+   A member sees the nodes of the forest that contain itself as a recipient as well as all descendants of such nodes;
```
Comment (marciw-da) [inline docs/2.8.0/docs/canton/architecture/overview.rst:491]:
```suggestion
   A member sees the nodes of the forest that contain itself as a recipient (as well as all descendants of such nodes),
```

File: docs/2.8.0/docs/canton/architecture/overview.rst [hunk 5]
```diff
@@ -537,13 +502,13 @@ participants, while preserving the participants' privacy,
...
+- collects mediator responses from participants,
 - validates them according to the Canton protocol,
-- computes the conclusions (approve / reject / timed out) according to the confirmation policy, and
+- computes the mediator verdict (approve / reject / timed out) according to the confirmation policy, and
 - sends the result message.
 
-Additionally, for auditability, the mediator persists every received
-message (containing informee information or confirmation responses) in long term storage
+Additionally, for auditability the mediator persists every finalized request
+together with its verdict in a long term storage
```
Comment (marciw-da) [inline docs/2.8.0/docs/canton/architecture/overview.rst:511]:
```suggestion
together with its verdict in long-term storage
```

File: docs/2.8.0/docs/canton/architecture/overview.rst [hunk 6]
```diff
@@ -563,101 +528,49 @@ A canonical example is a participant run by a trusted market operator.
...
+It uses the sequencer client to send the confirmation request to the mediator and all participants involved in the transaction.
 
-Identity Client
-~~~~~~~~~~~~~~~
+The transaction processor also uses the sequencer client to receive confirmation requests from the domain,
+to send mediator responses, and to receive the result messages from the mediator.
 
-The identity client handles the messages coming from the domain topology manager, and verifies the validity of the
-received identity information changes (for example, the validity of public key delegations).
+The multi domain event log stores for every request if has been committed vs. rolled back.
```
Comment (marciw-da) [inline docs/2.8.0/docs/canton/architecture/overview.rst:553]: Words missing here? Stores *what* for every request? "if has been committed" -- if *it* has been committed? 

Also "multi domain" needs a hyphen: "multi-domain"

Maybe it's this:
```suggestion
The multi-domain event log stores every committed request (but not rolled-back requests).
```

**Review by marciw-da — COMMENTED** [review]
Several suggestions -- happy to re-review!

File: docs/2.8.0/docs/canton/architecture/overview.rst [hunk 7]
```diff
@@ -563,101 +528,49 @@ A canonical example is a participant run by a trusted market operator.
...
-This implementation is further broken down
-into multiple components. We now describe the interface and
-properties of each component. The following figure shows the
-interaction between the different components and the relation to the
-existing Ledger API’s command and event services.
-
-.. https://www.lucidchart.com/documents/edit/8f0e701c-ca0a-4b59-8426-43fb3f4d0de2
+The below diagram shows the main components of a participant.
+
+.. https://lucid.app/lucidchart/effe03fe-e952-49e2-abc7-ac7a4cc56211/edit?page=0_0&invitationId=inv_f3433f52-c2ec-4854-9b36-58942657f191#
```
[the remark this answers is not in the document]
Comment (marciw-da) [reply docs/2.8.0/docs/canton/architecture/overview.rst:533]: whoops, misunderstood. Resolving

File: docs/2.8.0/docs/canton/architecture/overview.rst [hunk 8]
```diff
@@ -563,101 +528,49 @@ A canonical example is a participant run by a trusted market operator.
...
-When the other participants or domain entities do not behave according to the protocol (for example, not sending
-timely confirmation responses, or sending malformed requests), the transaction processing component raises alarms.
-
-**Confirmation Result Processing**. Based on the result message
-from the mediator, the transaction component commits or aborts
-the requested transaction.
-
-Sequencer Client
-~~~~~~~~~~~~~~~~
+A ledger application uses the ledger API (at the top of the diagram) to send commands to a participant and
```
[the remark this answers is not in the document]
Comment (matthiasS-da) [reply docs/2.8.0/docs/canton/architecture/overview.rst:539]: done

File: docs/2.8.0/docs/canton/architecture/overview.rst (hunk 6 above)
Comment (matthiasS-da) [reply docs/2.8.0/docs/canton/architecture/overview.rst:554]: I couldn't apply this as is, because is also stores rollbacks.

Comment (marciw-da) [reply docs/2.8.0/docs/canton/architecture/overview.rst]: Yep, I said "maybe" because I wasn't sure I understood... 🙂 Updated my suggestion.

**Pull request merged**